config/settings.yml

```
@@ -1,5 +1,6 @@
 header_logo_path: ~
 display_app_name_in_header: true
+stage_color: ~
```

Let's read from ENV here, where devs might expect to find an option like this.
Can you test what happens when no value is set?
```suggestion
stage_color: <%= ENV.fetch("STAGE_COLOR", nil) %>
```